src/PackageDependencyViewExtension/PackageDependencyView.xaml.cs

```diff
@@ -65,26 +65,18 @@ internal void DependencyRegen(WorkspaceModel ws)
             table.Columns.Clear();
             foreach (var package in ws.PackageDependencies)
             {
-                bool matchFound = false;
-                // Check if the target package is installed and loaded.
-                foreach (var loadedPackage in ws.LoadedPackageDependencies)
+                if (package.IsLoaded)
```

Comment:
Much easier to refresh UI now, great. The only question is that how costly to maintain this property on each node add operation, would love to know how slow it is